exporter/stackdriverexporter/config_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stackdriverexporter
+
+import (
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/open-telemetry/opentelemetry-service/config"
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
+	"github.com/open-telemetry/opentelemetry-service/exporter"
+)
+
+var _ = config.RegisterTestFactories()
```

Comment:
After this https://github.com/open-telemetry/opentelemetry-service/pull/159 is implemented we got rid of global factory registries and explicitly pass factories as dependencies to relevant functions (e.g. LoadConfigFile). Please see https://github.com/open-telemetry/opentelemetry-service/pull/159 for examples of how unit tests look like now and change this one to reflect the new approach.

Reply:
Updated and used the latest otel-svc in the dependencies.